kong/clustering/data_plane.lua

```diff
@@ -74,11 +74,20 @@ function _M.new(clustering)
 end
 
 
+local function set_control_plane_connected(reachable, ttl)
+  local ok, err = ngx.shared.kong:safe_set("control_plane_connected", reachable, ttl)
```

Comment:
I think we should just hard-code the ttl as `PING_WAIT` since it's what we should use in all cases.

```suggestion
local function set_control_plane_connected(reachable)
  local ok, err = ngx.shared.kong:safe_set("control_plane_connected", reachable, PING_WAIT)
```